spec/models/form_profile_spec.rb

```diff
@@ -12,6 +12,7 @@
 
   before do
     stub_evss_pciu(user)
+    user.instance_variable_set(:@ssn, user.ssn.delete('-'))
```

Comment:
What if the user or ssn is null? Will that cause any issues?

Reply:
Yep good call

Reply:
I added that line to resolve test failures, but the solution was elsewhere, so I am removing that addition